templates/web/bromley/header_logo.html

```diff
@@ -0,0 +1,62 @@
+    <div class="bromley-header cf">
+        <a href="http://www.bromley.gov.uk/" id="site-logo">[%- PROCESS 'site-name.html' -%]</a>
+
+        <ul class="linear-links subtext float-right top-links">
+            <li class="first-child"><i class="fa fa-home"></i><a href="http://www.bromley.gov.uk">Home</a></li>
+            <li><i class="fa fa-sitemap"></i><a href="http://www.bromley.gov.uk/site_map">Sitemap</a></li>
+            <li><i class="fa fa-user"></i><a href="https://www.bromley.gov.uk/SiteLogin">Register/Sign in</a></li>
+        </ul>
+
+        <form action="http://www.bromley.gov.uk/site/scripts/google_results.aspx" method="get" id="search" class="main-search" role="search">
+            <label for="SearchSite" class="hidden">Search this site</label>
+            <input type="text" size="18" class="main-search-input field" name="q" id="SearchSite" value="" placeholder="Search the website">
+            <input value="Search" type="submit" class="button__primary">
+        </form>
+    </div>
+</div>
+<div class="a-z-container ">
+    <nav class="main-a-z custom-side-nav">
+        <ul class="drop-down-nav">
+            <li class="first-child">
+                <a href="#" class="main-nav-title">A-Z</a>
+                <ul class="nav-hover-item nav-hover-item-active a-z" style="display: none;">
```

Comment:
This display:none should be added by the JS (or ideally, CSS using a .js class or similar for instant effect), as currently I don't think this works without JavaScript (though the comment in the a-z JS implies it should).